experimental/ExtendTeamsforM365/client/styles/northwind-teams.css

```diff
@@ -228,15 +228,18 @@ button#btnTaskModule {
 }
 
 div#allOrders {
-  width: 55%;
+  width: 58%;
 }
 div#allOrders, div#rODiv {
-  width: 55%;
+  width: 58%;
+}
+div#allOrders, div#rODiv {
+  width: 59%;
 }
 
 div#content {
   background: #ddd3;
-  padding: 10px;
+  padding: 10px;  
```

Comment:
drop trailing whitespace. after ;.

Reply:
Should detect mobile devices and change width to 100% on mobile and devices for div#allOrders ? @bhups-83

Reply:
Changes done for white space.